src/frontend/posts_page/components/TextInputEditor.tsx

```diff
@@ -128,6 +135,39 @@ class TextInputEditor extends React.Component<ITextareaProps, ITextareaState> {
         return options;
     };
 
+    addEmoji = (emoji: any) => {
+        this.setState({message: this.state.message + emoji})
+    };
+
+    showEmojiDropdown = () => {
+        this.setState({emojiDropDown: !this.state.emojiDropDown});
+    };
+
+    uploadFileOnChange = (event: any) => {
+        this.setState( {uploadedFiles: event.target.files});
+        let previewList = [];
+        for (let file of event.target.files) {
+            let preview =  URL.createObjectURL(file);
+            previewList.push(preview);
+        }
+        this.setState( {filesPreview: previewList});
+    };
+
+    uploadFileSubmit = async () => {
+        let fileList: IUploadedFile[] = [];
+        for (let image of this.state.uploadedFiles) {
+            let imageFileData = new FormData();
+            imageFileData.append('file', image);
+            let responsePost = await fetch('/api/v1/users/uploadAvatar',
+                {
+                    method: 'POST',
+                    body: imageFileData});
+            let responsePostData = await responsePost.text();
+            fileList.push({fileType: "image", path: responsePostData})
```

Comment:
Is fileType only "image"?

Reply:
I want it to be as simple as just "image" so that we don't need to specify the file type of image for now.